### ISSUE TITLE
Single shell script

### ISSUE THREAD

**Issue by smo-key** [lasarobotics/FTCVision-Training#2]
Can you write a single script to do everything at once, including build the opencv create_samples binary if it was not already built?

**Comment by ArnavMohan** [comment]
Sorry about that

**Comment by WSlatton** [comment]
Sure, but I think that I should keep the training script separate from the preparation ones as it can take a while to run.

**Comment by smo-key** [comment]
Nah, just have it all in one script, but if the script detects that preparation had already completed, then  have it prompt the user (re-run preparation stage? [y/N]).

**Comment by WSlatton** [comment]
Sounds good. I should have time to do it tomorrow.
